docs/source/markdown/podman-build.1.md

```diff
@@ -99,7 +99,7 @@ different stages in COPY instruction.
 
 Valid values could be:
 
-* Local directory – e.g. --build-context project2=../path/to/project2/src
+* Local directory – e.g. --build-context project2=../path/to/project2/src (This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
```

Comment:
I am not sure if this is true.  In certain circumstances, this should work on a MAC, as long as the baths on the MAC and in the VM Match.  If we volume mount in $HOME:$HOME and you specify --buildcontext=$HOME/foobar, it should work.

Reply:
Yes but in such context user has to do extra `--volume $HOME:$HOME` ideally this should happen automatically that's why i added a note that this is not available out of the box but I agree this works if users wants to do it manually.

Reply:
podman machine on MAC defaults to $HOME:$HOME

Reply:
oh then should i amend this to since everything on $HOME will be available inside machine.
```diff
-(This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
+(This option is not available with the remote Podman client)
```

Reply:
Yes, if you want to mention MAC, you could say that the PATH must be inside of the VM, not just on the host machine.

Reply:
updated.